### ISSUE TITLE
v3.0.1 download is v3.0.0

### ISSUE THREAD

**Issue by spoonwzd** [artman41/BeatSaverDownloaderApp#2]
As the title says, when I download v3.0.1 and run it, it still reports as v3.0.0 (and even brings up a new message saying v3.0.1 is available for download).

**Comment by jeremys18** [comment]
Same

**Comment by jeremys18** [comment]
This issue is over a year old. Is it going to be fixed or not? The app won't even work. Crashes as soon as I hit ok on the above message.

**Comment by whitetxt** [comment]
I'm guessing this isn't going to be fixed then.

**Comment by jeremys18** [comment]
I don't think this will ever be fixed. Don't worry though. I created my own downloader that downloads every song and it actually works. :)

**Comment by whitetxt** [comment]
Ah ok, I'll take a look at yours then. Thanks!
